assign_rights/models.py

```diff
@@ -40,24 +40,14 @@ def get_absolute_url(self):
         return reverse("rights-detail", kwargs={"pk": self.pk})
 
     def __str__(self):
-        start = "0 years after creation"
-        end = "open"
         prefixes = [self.get_rights_basis_display()]
         if self.rights_basis == "Copyright":
             prefixes.append(self.copyright_status)
         elif self.rights_basis == "License":
             prefixes.append(self.license_terms)
         elif self.rights_basis == "Statute":
             prefixes.append(self.statute_citation)
-        if self.start_date:
-            start = self.start_date
-        elif self.start_date_period:
-            start = "{} years after creation".format(self.start_date_period)
-        if self.end_date:
-            end = self.end_date
-        elif self.end_date_period:
-            end = "{} years after creation".format(self.end_date_period)
-        return "{} ({} until {})".format(" / ".join([p for p in prefixes if p]), start, end)
+        return "{} - {} ({})".format(self.pk, " / ".join([p for p in prefixes if p]), self.note)
```

Comment:
It might make sense to trim the note to a certain length and then append ellipses if the note is longer than that length. You may want to see what this looks like in the UI first though.